pallene/checker.lua

```diff
@@ -349,14 +306,60 @@ function Checker:check_program(prog_ast)
                 end
             end
 
+        elseif group_kind == "Stat" then
+            -- skip
+
         else
             error("impossible")
         end
     end
 
+    local total_nodes = #tls
+    if total_nodes == 0 then
+        return prog_ast
+    end
+
+    for i = 1, total_nodes - 1 do
+        local stat = tls[i].stat
+        if stat then
```

Comment:
This `if` statement should check the `_tag` of the `tls[i]` instead of testing if the `stat` field is present.